File: system/src/main/java/com/cys/system/common/service/OrderService.java
package com.cys.system.common.service;

import com.cys.system.common.common.pojo.Result;
import com.cys.system.common.common.pojo.UpOrder;
import com.cys.system.common.pojo.Order;

import java.util.List;

public interface OrderService {
    Result buildOrder(Integer[] ids) throws NoSuchFieldException;

    Result lookNoPay(List<String> noPayList);

    Result lookPay(Integer status,Integer userId,String orderId);

    Result pay(UpOrder upOrder) throws NoSuchFieldException;

    Result list(Integer page, Integer rows, Order order);

    Result updateStatus(Integer status, String[] ids,Integer userId,Integer shopId);

    Result findOrderItemByOrderId(Integer id);

    Result deleteOrder(String payToken);
}

File: system/src/main/resources/static/system/js/service/orderService.js
app.service("orderService", function ($http) {

    // 根据id查询
    this.findOrderItemByOrderId = function (id) {
        return $http.get(basePath+"/order/findOrderItemByOrderId?id="+id);
    };
    // 条件查询
    this.search = function (page, rows, searchEntity) {
        return $http.post(basePath+"/order/list?page=" + page + "&rows=" + rows,
            searchEntity);
    };
    this.updateStatus = function (status, ids) {
        return $http.put(basePath+"/order/updateStatus?status="+status+"&ids=" + ids);
    };

});